docs/setup/settings.asciidoc

```diff
@@ -296,9 +307,114 @@ This value must be a whole number greater than zero. *Default: `"1000"`*
 Maximum number of documents loaded by each shard to generate autocomplete
 suggestions. This value must be a whole number greater than zero.
 *Default: `"100000"`*
-+
+
 NOTE: To reload the <<logging-settings, logging settings>>, send a SIGHUP signal to {kib}.
 
+// deprecated logging configuration settings
```

Comment:
I've split the legacy and current "settings" into two grouped sections to make it clear what is deprecated and what current settings are/will be supported in 8.0